src/Network/Receive/kRO/Sakexe_0.pm

```diff
@@ -2265,6 +2267,41 @@ sub party_dead {
 	}	
 }
 
+sub progress_bar_unit {
+	my($self, $args) = @_;
+	debug "Displays progress bar (GID: $args-{GID} time: $args-{time})\n";	
```

Comment:
` $args-{GID} and $args-{time}`

fix it to

` $args->{GID} and $args->{time}`